### PR TITLE
feat(synthetics): syn-python-selenium-2.0 runtime

### LINKED ISSUE #24704: aws-synthetics-alpha: missing latest python runtime definition (2.0)
### Describe the bug

selfexplanatory

### Expected Behavior

python selenium runtimes including 2.0 should be available

### Current Behavior

`import { Runtime } from '@aws-cdk/aws-synthetics-alpha';`

`Runtime.SYNTHETICS_PYTHON_SELENIUM_2_0,` is not available in SYNTHETICS_PYTHON_SELENIUM family 

### Reproduction Steps

new Canary(this, 'Canary', {
    schedule: Schedule.rate(Duration.minutes(15)),
    test: Test.custom({
        code: Code.fromAsset(__dirname),
        handler: 'index.handler',
    }),
    runtime: Runtime.SYNTHETICS_PYTHON_SELENIUM_2_0,
});

### Possible Solution

I can see the definitions in repository but they are not built into alpha package

### Additional Information/Context

_No response_

### CDK CLI Version

2.100.0 (build 60a5b2a)
"@aws-cdk/aws-synthetics-alpha": "^2.100.0-alpha.0",

### Framework Version

_No response_

### Node.js Version

v18.17.1

### OS

OSX 13.6 (22G120)

### Language

Typescript

### Language Version

TS 4.9.5

### Other information

_No response_

### LINKED ISSUE #27566: synthetics: add selenium runtime 2.0
### Describe the feature

Python selenium runtime 2.0 exists: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_python_selenium.html but is not in aws-cdk-lib/aws-synthetics.

### Use Case

Latest runtime

### Proposed Solution

_No response_

### Other Information

_No response_

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

cdk v2.100.0

### Environment details (OS name and version, etc.)

mac

### PR DESCRIPTION
**Pull request by frichtarik**
adding missing selenium runtime for synthetics package

Closes #27566.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-synthetics-alpha/lib/runtime.ts
Comment (kaizencc) [inline packages/@aws-cdk/aws-synthetics-alpha/lib/runtime.ts:1]: we're not gonna add anything to the alpha package, its deprecated. in fact i'm removing the entire package today.

**Review by kaizencc — CHANGES_REQUESTED** [review]
Thanks for the contribution @frichtarik! See my 1 comment, but also this PR needs an update to an existing integ test and then that integ test needs to be run and updated.

File: packages/@aws-cdk/aws-synthetics-alpha/lib/runtime.ts
Comment (frichtarik) [reply packages/@aws-cdk/aws-synthetics-alpha/lib/runtime.ts:1]: Hi @kaizencc, thanks for your input ... however IMHO it's very strange decision to "deprecate" package if there is no way forward meaning there isn't one currently published standard package we could switch to. Or do I miss something ?

Comment (frichtarik) [reply packages/@aws-cdk/aws-synthetics-alpha/lib/runtime.ts:1]: I see it now, thank you. 
I've limited the PR to non-alpha package, however i'm required by linter GHA to adjust integration tests ... can you please point a finger as I don't see any in the package, and i'm a bit afraid that creating full suite is a bit out of scope of my first ever contribution :D

Comment (kaizencc) [reply packages/@aws-cdk/aws-synthetics-alpha/lib/runtime.ts:1]: The integ tests live in `@aws-cdk-testing/framework-integ/test/aws-synthetics`

The information for how to run is [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#integration-tests). 

You probably need to add a line very similar to this and then run the integ tests:

https://github.com/aws/aws-cdk/blob/c445b8cc6e20d17e4a536f17262646b291a0fe36/packages/%40aws-cdk-testing/framework-integ/test/aws-synthetics/test/integ.canary.ts#L101

**Comment by frichtarik** [comment]
> @frichtarik if you can't/haven't run the integ tests, I can run them for you.

i'll be actually glad, because i tried to run them on my own but they failed every time on creating autodeleteobject lambda related to s3 with `Resource handler returned message: "Uploaded file must be a non-empty zip (Service: Lambda, Status Code: 400, Request ID: 62d4ae87-45fc-4285-8686-617db14fb806)"`

**Comment by kaizencc** [comment]
hmm interesting. this seems like a problem with our integ runner somewhere. i get the same error, but not when i deploy the stack with `cdk deploy`. i'll keep looking into it tomorrow

**Review by kaizencc — APPROVED** [review]

**Pull request merged**